draft-ietf-quic-transport.md

```diff
@@ -2357,15 +2357,22 @@ containing frames other than ACK or PADDING (an ack-eliciting packet; see
 since last receiving a packet.  Restarting when sending packets ensures that
 connections do not prematurely time out when initiating new activity.
 
-The value for an idle timeout can be asymmetric.  The value advertised by an
-endpoint is only used to determine whether the connection is live at that
-endpoint.  An endpoint that sends packets near the end of the idle timeout
-period of a peer risks having those packets discarded if its peer enters the
-draining state before the packets arrive.  If a peer could timeout within a
-Probe Timeout (PTO; see Section 6.3 of {{QUIC-RECOVERY}}), it is advisable to
-test for liveness before sending any data that cannot be retried safely.  Note
-that it is likely that only applications or application protocols will
-know what information can be retried.
+Each endpoint can advertise a different idle timeout value. An idle timeout
+indicates the connection will be abandoned when the timer ends or soon
+afterwards.  If a peer advertises a smaller idle timeout value than the
```

Comment:
Each endpoint advertises a different idle timeout, but the effective value is determined by taking the minimum of the two advertised values.  By announcing an idle timeout, endpoints commit to initiating an immediate close ({{immediate-close}}) if it abandons a connection prior to the effective value.